src/pages/Dragons.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getDragonData } from '../Redux/Dragons/dragonSlice';
+import DragonsCreator from '../components/DragonsCreator';
+import '../styles/dragonStyle.css';
+
+const Dragons = () => {
+  const dragons = useSelector((state) => state.dragons.dragons);
+  const theDispatch = useDispatch();
+
+  useEffect(() => {
+    theDispatch(getDragonData());
+  }, [theDispatch]);
```

Comment:
- [ ] Amazing job with the design and functionality. 💯️ There seems to be a small bug in this `useEffect` function. Every time I visit this page with the navigation, it is dispatching an action to fetch data from the API which in turn replaces the current state we had. You did a great job of preventing this issue on the Missions and Rockets page. Kindly implement the same thing here. ✍️

Reply:
thanks for the tip